### PR TITLE
16.1 update

### LINKED ISSUE #1366: Include AWX (or other UI) in Ansible appliance
Mike (via the [forums](https://www.turnkeylinux.org/forum/general/wed-20191023-1346/ansible-awx)) has requested we include [AWX](https://github.com/ansible/awx) in our [Ansible appliance](https://www.turnkeylinux.org/ansible).

AWX is the open source (Apache licensed) project that underlies [Ansible Tower](https://www.ansible.com/products/tower). AWX provides a WebUI, REST API, and task engine for Ansible.

I think it would be a fantastic inclusion! :smile: Install instructions are [here](https://github.com/ansible/awx/blob/devel/INSTALL.md).

### PR DESCRIPTION
**Pull request by JedMeister**
Major reworking/refactoring of our Ansible app. Primarily to include the Semaphore web UI.

The inclusion of Semaphore UI closes https://github.com/turnkeylinux/tracker/issues/1366

### CODE REVIEW

**Comment by JedMeister** [comment]
@ongle

File: changelog [hunk 1]
```diff
@@ -23,7 +41,7 @@ turnkey-ansible-15.1 (1) turnkey; urgency=low
   * Include Debian 'python3-pyfits' package. Not directly required but
     includes a working weakref.py file. The Python3 version in Stretch is
     buggy - see Debian bug #918716. Closes #1324.
-    Issue #1325 tracks the future removal of the unrequired dependency at some
+    Issue #1325 tracks the future removal of the unrequited dependency at some
```
Comment (OnGle) [inline changelog]: Is this change intentional?

**Review by OnGle — APPROVED** [review]
Awesome one man!

File: changelog (hunk 1 above)
Comment (JedMeister) [reply changelog]: Haha! Great catch, no it wasn't... I'll fix that! :smile:

**Comment by JedMeister** [comment]
@OnGle - I fixed the issue you noted, But I also spotted another oversight (use of an unassigned variable... doh!) so fixed that too!

**Review by OnGle — APPROVED** [review]
lgtm!

**Comment by OnGle** [comment]
Awesome! Good catch!
